src/main/java/com/google/firebase/auth/FirebaseTokenVerifierImpl.java

```diff
@@ -97,7 +99,7 @@ private FirebaseTokenVerifierImpl(Builder builder) {
   public FirebaseToken verifyToken(String token) throws FirebaseAuthException {
     IdToken idToken = parse(token);
     checkContents(idToken);
-    if (!isEmulatorMode()) {
+    if (!isEmulatorMode) {
```

Comment:
If you want you can make `isEmulatorMode` a local variable here instead of a class attribute.

Reply:
Done. Thanks!